### PR TITLE
path can be empty string

### PR DESCRIPTION
**Pull request by sunnylqm**

### CODE REVIEW

**Comment by chimurai** [comment]
Please provide a description and test plan for this PR.

**Comment by sunnylqm** [comment]
@chimurai for example
```
{
        path: '/proxy',
        target: 'http:/example.com',
        pathRewrite: () => ''
}
```

If I want to replace the whole path into an empty string, then it would complain "No rewritten path found"

File: lib/index.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ function HttpProxyMiddleware(context, opts) {
         if (pathRewriter) {
             var path = pathRewriter(req.url, req);
 
-            if (path) {
+            if (path !== false) {
```
Comment (chimurai) [inline lib/index.js]: Change the condition to:

```javascript
if (typeof path === 'string') {
   ...
}
```

**Comment by sunnylqm** [comment]
Can it be merged now @chimurai

**Comment by chimurai** [comment]
Sorry for the late response.
Thanks for contributing!

**Pull request merged**